code/data_structures/queue/reverse_queue/Reverse_queue.c

```diff
@@ -0,0 +1,136 @@
+#include <stdio.h>
+#include <stdlib.h>
+#define SIZE 5
+
+struct tag
+{
+        int a[SIZE];
+        int top;
+};   //Structure definition for stack
+typedef struct tag stack;
+
+
+struct tagg
+{
+        int items[SIZE];
+        int rear;
+        int front;
+};  //Structure definition for queue
+typedef struct tagg QUEUE;
+
+
+void push();
+int pop();
+void display();
+void rev(); //Function declarations(parameters not required at declaration)
+
+
+int main()
+{
+        int ch, element, dnum;
+        stack st;
+        st.top= -1;
+ 
+        do
+        {       //Displaying menu for operations on stack
+                printf("\n1.) PUSH");
```

Comment:
Kindly define the input within the code instead of using user input in a switch case 👍